src/game/character/character.gd

```diff
@@ -3,7 +3,7 @@ extends "res://game/entity.gd"
 # --Public Variables--
 onready var abilityPoint = $CharacterElements/Abilities
 onready var itemIntArea = $CharacterElements/ItemInteraction
-onready var taskIntArea = $CharacterElements/TaskInteraction
```

Comment:
Removing this actually breaks the chemical cabinet, but for some reason it doesn't break the task pick up from the floor?!